x-pack/test_serverless/functional/test_suites/common/discover/x_pack/reporting.ts

```diff
@@ -132,25 +124,29 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       it('generates a report with no data', async () => {
         await PageObjects.reporting.setTimepickerInEcommerceNoDataRange();
+        await PageObjects.discover.clickNewSearchButton();
         await PageObjects.discover.saveSearch('my search - no data - expectReportCanBeCreated');
 
         const res = await getReport();
         expect(res.text).to.be(`\n`);
       });
 
       it('generates a large export', async () => {
+        await PageObjects.discover.clickNewSearchButton();
         const fromTime = 'Apr 27, 2019 @ 23:56:51.374';
         const toTime = 'Aug 23, 2019 @ 16:18:51.821';
         await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
-        await PageObjects.discover.clickNewSearchButton();
+        await PageObjects.unifiedFieldList.clickFieldListItemAdd('order_id');
+        await PageObjects.discover.clickFieldSort('order_id', 'Sort A-Z');
+
         await retry.try(async () => {
           expect(await PageObjects.discover.getHitCount()).to.equal('4,675');
         });
         await PageObjects.discover.saveSearch('large export');
 
         // match file length, the beginning and the end of the csv file contents
         const { text: csvFile } = await getReport({ timeout: 80 * 1000 });
-        expect(csvFile.length).to.be(4845684);
+        expect(csvFile.length).to.be(171599);
         expectSnapshot(csvFile.slice(0, 5000)).toMatch();
         expectSnapshot(csvFile.slice(-5000)).toMatch();
```

Comment:
I wonder if it makes sense to test this way, given we have just 4675 matching records

Reply:
@tsullivan  Am I overseeing something (I'm trying to fix a serverless flakiness here, but I want to understand why it's written this way)

Reply:
nvm, it's checking the first 5000 characters, and the last 5000 characters of the file ... not the lines :D ... I didn't pass the coding test 😁